requirements/base.txt

```diff
@@ -69,7 +69,7 @@ cryptography==44.0.1
     #   mozilla-django-oidc
     #   pyopenssl
     #   webauthn
-django==4.2.19
+django==4.2.20
```

Comment:
@Floris272 could you also bump jinja2 to fix this security vulnerability? https://github.com/maykinmedia/objecttypes-api/security/dependabot/371